### PR TITLE
fix(typo): link to spock not well written

### PR DESCRIPTION
**Pull request by NicolasGeraud**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -324,7 +324,7 @@ Standard Maven build:
 
 == Testing
 
-http://spockframework.org/(Spock) is used for testing the calling of the Mojo. This will be downloaded by Maven. Tests are run simply by:
+link:http://spockframework.org/[Spock] is used for testing the calling of the Mojo. This will be downloaded by Maven. Tests are run simply by:
```
Comment (mojavelinux) [inline README.adoc]: The `link:` part isn't necessary. Simply changing the round brackets to square brackets will do it.

**Comment by NicolasGeraud** [comment]
ok, what do you prefer : I force push the correct commit or just push the fix's fix ?

**Comment by mojavelinux** [comment]
Force push the change to remove the `link:` prefix. Thanks!

**Comment by NicolasGeraud** [comment]
et voilà

**Comment by mojavelinux** [comment]
Merci !

**Comment by NicolasGeraud** [comment]
I've just have a look at the error reported by appveyor.
https://ci.appveyor.com/project/asciidoctor/asciidoctor-maven-plugin/build/45/job/qxuobaw0aoj7kiyu

It seems that the error is not due to my commit.

**Comment by LightGuard** [comment]
The http stuff in the maven plugin has threading issues. We really should
find out who uses it and pull it out if it isn't being used. Or someone
needs to fix the tests that's better at threading :)

On Tue, Sep 22, 2015 at 2:07 PM, Nicolas Géraud <notifications@github.com>
wrote:

> I've just have a look at the error reported by appveyor.
>
> https://ci.appveyor.com/project/asciidoctor/asciidoctor-maven-plugin/build/45/job/qxuobaw0aoj7kiyu
>
> It seems that the error is not due to my commit.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/asciidoctor/asciidoctor-maven-plugin/pull/169#issuecomment-142404031>
> .
>



-- 
Jason Porter
http://en.gravatar.com/lightguardjp

**Comment by NicolasGeraud** [comment]
@LightGuard I could modify the test and use a random available port instead of using the default one.

What do you think of that ?

**Comment by LightGuard** [comment]
Uh, that may work well enough. Nothing immediately comes to mind why it
wouldn't work.

On Tue, Sep 22, 2015 at 2:44 PM, Nicolas Géraud <notifications@github.com>
wrote:

> @LightGuard <https://github.com/LightGuard> I could modify the test and
> use a random available port instead of using the default one.
>
> What do you think of that ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/asciidoctor/asciidoctor-maven-plugin/pull/169#issuecomment-142414989>
> .
>



-- 
Jason Porter
http://en.gravatar.com/lightguardjp

**Comment by mojavelinux** [comment]
@NicolasGeraud If you could sort out how to fix that test, that would really help us out. But I won't cause it to hold up this merge.

**Comment by mojavelinux** [comment]
@NicolasGeraud Could you file a separate issue to track the fix to the test on appveyor?

**Comment by NicolasGeraud** [comment]
@mojavelinux

**Pull request merged**